modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
+.Prerequisites
```

Comment:
The prereqs list should be bulleted rather than ordered

Reply:
Updated the content - prerequisites are bulleted.